packages/kbn-apm-config-loader/src/utils/get_config_file_paths.ts

```diff
@@ -22,5 +22,9 @@ export const getConfigurationFilePaths = (argv: string[]): string[] => {
   if (rawPaths.length) {
     return rawPaths.map((path) => resolve(process.cwd(), path));
   }
-  return [getConfigPath()];
+
+  const configPath = getConfigPath();
+
+  // Pick up settings from dev.yml as well
+  return [configPath, configPath.replace('kibana.yml', 'kibana.dev.yml')];
```

Comment:
If anyone has a better idea how to get configs from `kibana.dev.yml` in a package, I'm open to it!